zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterServer.java

```diff
@@ -140,10 +136,9 @@ public static void main(String[] args)
 
 
   @Override
-  public void createInterpreter(String interpreterGroupId, String className, Map<String, String>
-          properties)
-      throws TException {
-
+  public void createInterpreter(String interpreterGroupId, String noteId, String
```

Comment:
Sure, there's no problem in this implementation. And I agree with yours of updating interpreter api. Go forward first.